Dockerfile

```diff
@@ -7,21 +7,7 @@ ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
```

Comment:
this is not needed?

Reply:
I think it is. I’ll add this back and bring that to Jobset.

Reply:
updated.